test/DomainRegistry.test.cjs

```diff
@@ -0,0 +1,98 @@
+const { expect } = require('chai');
+const { ethers } = require('hardhat');
+
+describe("DomainRegistry", () => {
+  let DomainRegistry;
+  let domainRegistry;
+  let owner;
+  let addr1;
+  let addr2;
+  let addrs;
+
+  beforeEach(async () => {
```

Comment:
/// tests 
// line 12 
// as best practice it is better to use before hook to one time deploy the contract and use it in all tests.